test/python/circuit/library/test_nlocal.py

```diff
@@ -415,6 +415,31 @@ def test_entanglement_by_list(self):
 
         self.assertCircuitEqual(nlocal, circuit)
 
+    def test_initial_state_as_QuantumCircuit_object(self):
```

Comment:
We usually use snake_case exclusively for the method names 🙂 
```suggestion
    def test_initial_state_as_circuit_object(self):
```